packages/@aws-cdk/toolkit-lib/lib/actions/diff/index.ts

```diff
@@ -129,4 +129,11 @@ export interface DiffOptions {
    * @deprecated implement in IoHost
    */
   readonly securityOnly?: boolean;
+
+  /**
+   * Whether or not the change set imports resources that already exist
+   *
+   * @default false
+   */
+  readonly importExistingResources?: boolean;
```

Comment:
This needs to go into `method` and `DiffMethod` (and ultimately `ChangeSetDiffOptions`) because it's only available for changesets. See deploy action and `ChangeSetDeploymentMethod`.